#! /bin/sh /usr/share/dpatch/dpatch-run
## testsuite-increase-wait-timeout.dpatch by Martin Pitt <mpitt@debian.org>
##
## DP: Increase test suite's timeout for waiting for jobs, for slower architectures like arm and m68k
## DP: (http://bugs.debian.org/518787)

@DPATCH@
diff -urNad cups-1.3.9~/test/waitjobs.sh cups-1.3.9/test/waitjobs.sh
--- cups-1.3.9~/test/waitjobs.sh	2008-03-29 01:34:23.000000000 +0100
+++ cups-1.3.9/test/waitjobs.sh	2009-03-10 13:27:44.776659247 +0100
@@ -17,7 +17,7 @@
 if test $# = 1; then
 	timeout=$1
 else
-	timeout=60
+	timeout=600
 fi
 
 echo "Waiting for jobs to complete..."
